src/math/decibels.c

```diff
@@ -105,10 +106,10 @@ int32_t exp_fixed(int32_t x)
 		n++;
 	}
 
-	/* exp_small_fixed() input is Q3.29, while x1 is Q5.27
-	 * exp_small_fixed() output is Q9.23, while y0 is Q12.20
+	/* sofm_exp_int32() input is Q4.28, while x1 is Q5.27
```

Comment:
for comments, you mentioned input is Q4.28, however, what is x1? means sofm_exp_int32() output?
if means output, please change x1 to output.